### PR TITLE
Fix minimal panel close button on next line

### LINKED ISSUE #1979: Minimal panel button wrapper is on the next line
### Please confirm that you have searched existing issues in the repo

Yes, I have searched the existing issues

### Any related issues?

_No response_

### Tell us about your environment

Anything

### MarkBind version

4.0

### What did you do? Describe the bug

Untitled

Probably a regression from: https://github.com/MarkBind/markbind/pull/1694

In particular:
- https://github.com/MarkBind/markbind/pull/1694#pullrequestreview-860602847 (cc @jonahtanjz for context on what it was trying to solve if any, we should try to preserve both desired behaviours)
- https://github.com/MarkBind/markbind/pull/1694/commits/b435cc3b53c8224872d0abdad3e05b8ef640f677

### Steps to reproduce

_No response_

### Expected behavior

_No response_

### Actual behavior

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by jonahtanjz**
**What is the purpose of this pull request?**

- [ ] Documentation update
- [x] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:

**Overview of changes:**
Fixes #1979

Add `display: inline-block` to `button-wrapper` class so that the close button will be on the same line. Adjusted the `top` values of `minimal-close-button` and `minimal-menu-up` as well to align them together.

After:
image

**Anything you'd like to highlight/discuss:**
NIL

**Testing instructions:**
1. Go to https://deploy-preview-2000--markbind-master.netlify.app/userguide/components/presentation#panels
2. Scroll down to the minimal panel example
3. Expand minimal panel and then click on the up caret icon
4. Hover over to to check close button

**Proposed commit message: (wrap lines at 72 characters)**
Fix minimal panel close button on next line

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No unrelated changes

### CODE REVIEW

File: packages/vue-components/src/panels/NestedPanel.vue [hunk 1]
```diff
@@ -268,6 +268,7 @@ export default {
     }
 
     .button-wrapper {
+        display: inline-block;
```
Comment (tlylt) [inline packages/vue-components/src/panels/NestedPanel.vue:271]: I am seeing a warning about the use of `display:inline-block` together with `float:right`, wonder if this is ok?
image
https://developer.mozilla.org/en-US/docs/Web/CSS/float

File: packages/vue-components/src/panels/NestedPanel.vue [hunk 2]
```diff
@@ -355,6 +356,7 @@ export default {
         }
 
         .button-wrapper {
+            display: inline-block;
```
Comment (tlylt) [inline packages/vue-components/src/panels/NestedPanel.vue:359]: I think properties are inherited if not specified, so we may not need to include this in the media query. Was it included for easy of maintenance or?

**Review by tlylt — COMMENTED** [review]
Thanks @jonahtanjz, functionalities wise the close button works as described. I left two comments for clarification pls.

File: packages/vue-components/src/panels/NestedPanel.vue (hunk 1 above)
Comment (jonahtanjz) [reply packages/vue-components/src/panels/NestedPanel.vue:271]: Yup seems like `display: inline-block` will override the `float: right` property for minimal panels. `float: right` is still needed for the buttons on the normal panels.

File: packages/vue-components/src/panels/NestedPanel.vue (hunk 2 above)
Comment (jonahtanjz) [reply packages/vue-components/src/panels/NestedPanel.vue:359]: It was included in the previous implementation and I just reverted it. Seems like this is not needed 😅 Anyway I have shifted this to the minimal panel component.

**Review by tlylt — APPROVED** [review]
LGTM!

**Pull request merged**